be/src/olap/delete_bitmap_calculator.h

```diff
@@ -82,31 +97,41 @@ class MergeIndexDeleteBitmapCalculatorContext {
     size_t _cur_row_id {0};
     size_t const _num_rows;
     size_t const _max_batch_size;
+    size_t const _end_version;
+    bool const _base_segment;
+    RowsetId const _rowset_id;
     int32_t const _segment_id;
-    bool _excat_match {false};
+    bool _excat_match;
 };
 
-class MergeIndexDeleteBitmapCalculator {
+class MergedPKIndexDeleteBitmapCalculator {
 public:
-    MergeIndexDeleteBitmapCalculator() = default;
-
-    Status init(RowsetId rowset_id, std::vector<SegmentSharedPtr> const& segments,
-                size_t seq_col_length = 0, size_t max_batch_size = 1024);
+    MergedPKIndexDeleteBitmapCalculator() = default;
 
-    Status calculate_one(RowLocation& loc);
+    Status init(std::vector<SegmentSharedPtr> const& segments,
```

Comment:
There's 2 ways of using `MergedPKIndexDeleteBitmapCalculator`, you need to explain how to use it for these 2 ways in comment.